Fix: format sets default reputation command wrong (#3038)
demisto/demisto-sdk: CHANGELOG.md

# Changelog

## Unreleased

> * Fixed an issue where **format** added default arguments even is exist already.

Reviewer: ```suggestion
* Fixed an issue where **format** added default arguments to reputation commands which already have one.
```
